tests/test_chebyMap.py

```diff
@@ -0,0 +1,493 @@
+            return (
+                -1.1 * cheby(2, x1) * cheby(0, x2) + 1.3 * cheby(3, x1) * cheby(1, x2),
+            )
+
+        # Coefficients for the following polynomial:
+        # y1 = 1.6 T3(x1')
+        # y2 = -3.6 T1(x1')
+        coeff_i = np.array([
+            [1.6, 1, 3],
+            [-3.6, 2, 1],
+        ])
+        self.assertEqual(nout, coeff_i.shape[1] - 2)
+
+        def referenceCheby_i(point):
+            """Reference inverse implementation; point is normalized
+            """
+            x1 = point
+            return (
+                1.6 * cheby(3, x1),
+                -3.6 * cheby(1, x1),
+            )
+
+        refCheby_f = ReferenceCheby(referenceCheby_f, lbnd_f, ubnd_f)
+        refCheby_i = ReferenceCheby(referenceCheby_i, lbnd_i, ubnd_i)
+
+        chebyMap = astshim.ChebyMap(coeff_f, coeff_i, lbnd_f, ubnd_f, lbnd_i, ubnd_i)
+        self.assertEqual(chebyMap.getNin(), 2)
+        self.assertEqual(chebyMap.getNout(), 1)
+
+        self.checkBasicSimplify(chebyMap)
+        self.checkCopy(chebyMap)
+        self.checkPersistence(chebyMap)
+
+        outPoints_f = chebyMap.tranForward(inPoints_f)
+        desiredOutPoints_f = refCheby_f.transform(inPoints_f)
+
+        npt.assert_allclose(outPoints_f, desiredOutPoints_f)
+
+        outPoints_i = chebyMap.tranInverse(inPoints_i)
+        desiredOutPoints_i = refCheby_i.transform(inPoints_i)
+
+        npt.assert_allclose(outPoints_i, desiredOutPoints_i)
+
+    def test_ChebyMapPolyTran(self):
+        nin = 2
+        nout = 2
+        lbnd_f = [-2.0, -2.5]
+        ubnd_f = [1.5, 2.5]
+
+        # arbitrary points that cover the input range
+        inPoints = np.array([
+            [-2.0, 0.0],
+            [-1.0, -2.5],
+            [0.1, -0.2],
+            [1.5, 2.5],
+            [1.0, 2.5],
+        ])
+
+        # Coefficients for the following gently varying polynomial:
+        # y1 = -2.0 T0(x1') T0(x2') + 0.11 T1(x1') T0(x2') - 0.2 T0(x1') T1(x2') + 0.001 T2(x1') T1(x2')
+        # y2 =  5.1 T0(x1') T0(x2') - 0.55 T1(x1') T0(x2') + 0.13 T0(x1') T1(x2') - 0.002 T1(x1') T2(x2')
+        coeff_f = np.array([
+            [-2.0, 1, 0, 0],
+            [0.11, 1, 1, 0],
+            [-0.2, 1, 0, 1],
+            [0.001, 1, 2, 1],
+            [5.1, 2, 0, 0],
+            [-0.55, 2, 1, 0],
+            [0.13, 2, 0, 1],
+            [-0.002, 2, 1, 2]
+        ])
+        self.assertEqual(nin, coeff_f.shape[1] - 2)
+
+        def referenceFunc(point):
+            """Reference implementation; point is normalized
+            """
+            x1, x2 = point
+            return (
+                -2.0 * cheby(0, x1) * cheby(0, x2) + 0.11 * cheby(1, x1) * cheby(0, x2) -
+                0.2 * cheby(0, x1) * cheby(1, x2) + 0.001 * cheby(2, x1) * cheby(1, x2),
+                5.1 * cheby(0, x1) * cheby(0, x2) - 0.55 * cheby(1, x1) * cheby(0, x2) +
+                0.13 * cheby(0, x1) * cheby(1, x2) - 0.002 * cheby(1, x1) * cheby(2, x2),
```

Comment:
I hadn't thought of trying `\`; being able to indent would have been worth that, I think (but that code is gone).